README.md

```diff
@@ -1,82 +1,147 @@
+[![JavaScript Style Guide: Good Parts](https://img.shields.io/badge/code%20style-goodparts-brightgreen.svg?style=flat)](https://github.com/dwyl/goodparts "JavaScript The Good Parts")
+
 
-An ESLint Style that _only_ allows JavaScript the ***Good Parts*** (*and "Better Parts"*) in your code you can sleep at night.
 
 ## *Why?*
 
-Because this is the _best_ way we know how to write consistent,
-predictable and error free JavaScript code.
+Using JavaScript "**The _Good_ Parts**" (_abreviated to **`"goodparts"`**_)
+is the _best_ way we know <br />
+how to write _simple_, _consistent_,
+_predictable_ and 100% cross-browser-compatible JavaScript code.
 
-Our ***guiding principal*** is:
+The ***guiding principal*** of `goodparts` is:
 
 > "***perfection*** *is* ***attained*** *not when there is nothing more to add,  
 > but when there is* ***nothing more to remove***" ~
 [Antoine de Saint-Exupéry](http://en.wikiquote.org/wiki/Antoine_de_Saint_Exup%C3%A9ry#Quotes)
 
-we subscribe to the "_code is for **humans** to read_" idea:
+We _agree_ that "_code is for **humans** to read_":
 
-> "_**Programs must be written for people to read**, and only incidentally for machines to execute._" ~ Harold Abelson
+> "_**Programs must be written for people to read**, <br />
+and only incidentally for machines to execute._" ~ Harold Abelson
 
-> "_Any fool can write code that a computer can understand.
+> "_Any fool can write code that a computer can understand. <br />
 **Good programmers write code** that **humans** can
 **understand**._" ~ Martin Fowler
 
-Also, we agree with:
-
-> "_**constraints** can actually **boost creativity**_". [Martin Villeneuve](https://www.ted.com/talks/martin_villeneuve_how_i_made_an_impossible_film)
-
 The 3 biggest advantages of using `goodparts` in your code:
 
 1. [***Readability***](https://en.wikipedia.org/wiki/Readability) - using _fewer_ ECMAScript (JS) language features means
 people reading your code can get to the heart of what it *does*
 rather than waste time trying to understand
 how its been written.
-2. [***Shoshin***](https://en.wikipedia.org/wiki/Shoshin) - using _fewer_ JS features
-also means a _complete_ beginner has _less_ to learn before
-they _understand_ your code. This is _essential_ in Open Source
-if you want to make something that _anyone_ can use/modify/extend.
-3. [***Portability***](https://en.wikipedia.org/wiki/Software_portability) -
+2. [***Portability***](https://en.wikipedia.org/wiki/Software_portability) -
 "Old" Browsers/Devices are a _fact_, even if it's "*only a small percentage*"
 of people, we _chose_ to
 [*empathize*](http://www.merriam-webster.com/dictionary/empathize)
 with them by writing code that ***Works Everywhere***<sup>TM</sup>.
-
+3. ***Beginner-friendlyness*** or [***Shoshin***](https://en.wikipedia.org/wiki/Shoshin) - using _fewer_ JS features
```

Comment:
Mis-spelling of `friendliness` but I plan to review the goodparts readme more fully in the coming days so I will correct then.